marklogic-data-hub-central/ui/e2e/cypress/integration/login/authorization.spec.tsx

```diff
@@ -48,136 +56,146 @@ describe('login', () => {
+      toolbar.getLoadToolbarIcon().click();
+      loadPage.loadView('th-large').should('be.visible');
+      loadPage.addNewButton('card').should('not.be.visible');
+      loadPage.stepSettings(stepName).click();
+      loadPage.stepNameInSettings().should('have.text', stepName);
+      loadPage.saveSettings(stepName).should('be.disabled');
+      loadPage.cancelSettings(stepName).click();
+      loadPage.editStepInCardView(stepName).click();
+      loadPage.saveButton().should('be.disabled');
+      loadPage.cancelButton().click();
+      loadPage.deleteStepDisabled(stepName).should('exist');
+      loadPage.stepName(stepName).trigger('mouseover');
+      loadPage.addToNewFlow(stepName).click();
+      runPage.newFlowModal().should('not.be.visible');
+      loadPage.existingFlowsList(stepName).click();
+      loadPage.existingFlowsList(flowName).should('not.be.visible');
+
+      loadPage.loadView('table').click();
+      tiles.waitForTableToLoad();
+      loadPage.addToFlowDisabled(stepName).should('exist');
+      loadPage.stepSettings(stepName).click();
+      loadPage.saveSettings(stepName).should('be.disabled');
+      loadPage.cancelSettings(stepName).click();
+      loadPage.stepName(stepName).click();
+      loadPage.saveButton().should('be.disabled');
+      loadPage.cancelButton().click();
+      loadPage.deleteStepDisabled(stepName).should('exist');
   });
 
-  it('should display appropriate tiles for hub-central-mapping-reader', () => {
-      cy.loginAsTestUserWithRoles('hub-central-mapping-reader').withUI()
+  it('should only enable Curate and Explorer tile for hub-central-mapping-reader', () => {
+      let entityTypeId = 'Customer'
+      let mapStepName = 'mapCustomersXML'
+      cy.loginAsTestUserWithRoles('hub-central-mapping-reader').withRequest()
           .url().should('include', '/tile');
       //All tiles but Explore and Model, should show a tooltip that says contact your administrator
       ['Load', 'Model', 'Run'].forEach((tile) => {
-          toolbar.getToolBarIcon(tile).trigger('mouseover');
-        //   cy.contains(`${tile}: Contact your security administrator to get the roles and permissions required to access this functionality.`);
-          cy.waitUntil(() => cy.contains(`${tile}: Contact your security administrator to get the roles and permissions required to access this functionality.`))
+          toolbar.getToolBarIcon(tile).should('have.attr', {style: 'cursor: not-allowed'})
       });
 
-      ['Curate', 'Explore'].forEach((tile) => {
-          toolbar.getToolBarIcon(tile).trigger('mouseover');
-        //   cy.contains(`${tile}`);
-          cy.waitUntil(() => cy.contains(`${tile}`))
-      });
-
-      //Mapping tests will verify that a valid user is able to access Curate tile and its features
+      toolbar.getCurateToolbarIcon().click();
+      curatePage.toggleEntityTypeId(entityTypeId);
+      curatePage.verifyTabs(entityTypeId, 'be.visible', 'not.exist');
```

Comment:
added this method to verify Map, Custom tabs in Curate tile that is exposed on roles. IDE will give a better picture of the params. Added docs to help understand the params, just mouse over the method name for more details in your IDE.